Dockerfile

```diff
@@ -41,9 +41,20 @@ ENV PATH ${PATH}:/home/.gimme/versions/go1.17.linux.arm64/bin:/home/.gimme/versi
 RUN go version
 
 WORKDIR /tmp/fluent-bit-$FLB_VERSION/
-RUN git clone https://github.com/matthewfala/fluent-bit.git /tmp/fluent-bit-$FLB_VERSION/
+RUN git clone https://github.com/fluent/fluent-bit.git /tmp/fluent-bit-$FLB_VERSION/
 WORKDIR /tmp/fluent-bit-$FLB_VERSION/build/
-RUN git fetch origin && git checkout 1_8_9_patch && git status
+RUN git fetch --all --tags && git checkout tags/v${FLB_VERSION} -b v${FLB_VERSION} && git describe --tags
+
+RUN git config --global user.email "falamatt@amazon.com" \
+  && git config --global user.name "Matthew Fala"
```

Comment:
I was getting errors when an email and name were not configured on git cherry picks. I entered my email for now, however if there is a better email to use like one that we use for the observability team or firelens team, I can change it to that.

Reply:
Can we use our Firelens team email?

Reply:
What is our firelens team email?

Reply:
aws-firelens@amazon.com

Reply:
great! Updating the pr.